### PR TITLE
fix: loading icon position

### LINKED ISSUE #10258: "Loading messages …" placed too low
### Steps to reproduce

1. Add an account
2. Click on a mailbox

### Expected behavior

Good design

### Actual behavior

Image

### Mail app version

4.1.0

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by keremsemiz**
Fixes #10258

### CODE REVIEW

**Comment by keremsemiz** [comment]
Screenshot 2024-10-17 150855

File: src/components/Mailbox.vue [hunk 1]
```diff
@@ -488,5 +488,9 @@ export default {
 }
 .empty-content {
 	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: flex-end;
+	margin-bottom: 100px;
```
Comment (ChristophWurst) [inline src/components/Mailbox.vue:494]: This *might* be problematic with different screen sizes. I suggest testing it with the responsive mode of the browser to see if it also looks good on mobile, tablets and larger screens :)

**Review by ChristophWurst — COMMENTED** [review]
Thank you :pray:

File: src/components/Mailbox.vue (hunk 1 above)
Comment (keremsemiz) [reply src/components/Mailbox.vue:494]: Thank you for reminding me! I completely forgot to test different screen sizes, I unfortunately cannot test them right now as I don't have my laptop with me, but I will test this and add media viewport tags if necessary or change the method I used in general :)

Comment (keremsemiz) [reply src/components/Mailbox.vue:494]: I just checked layout for all screen sizes, it works perfectly fine for really small and really large ones :)

**Review by GretaD — APPROVED** [review]

**Pull request merged**